### ISSUE TITLE
Regression in v0.13.2 when running puppeteer, still a problem in v0.15.0

### ISSUE THREAD

**Issue by styfle** [zeit/ncc#280]
## Steps to Reproduce

```sh
git clone https://github.com/styfle/og-image
cd og-image
ncc build src/card.ts
```

## Error

```
ERROR in ./node_modules/chrome-aws-lambda/source/index.js 167:6-67
Module not found: Error: Can't resolve '/Users/styfle/Code/zeit/og-image/node_modules/chrome-aws-lambda/source/source/puppeteer/lib' in '/Users/styfle/Code/zeit/og-image/node_modules/chrome-aws-lambda/source'
```

<details>
<summary>Click for full output</summary>
<pre>
ncc: Using typescript@3.2.4 (local user-provided)
ncc: Module directory "/Users/styfle/Code/zeit/og-image/node_modules/debug/src" attempted to require "supports-color" but could not be resolved, assuming external.
ncc: Module directory "/Users/styfle/Code/zeit/og-image/node_modules/ws/lib" attempted to require "bufferutil" but could not be resolved, assuming external.
ncc: Module directory "/Users/styfle/Code/zeit/og-image/node_modules/ws/lib" attempted to require "utf-8-validate" but could not be resolved, assuming external.
ncc: Module directory "/Users/styfle/Code/zeit/og-image/node_modules/https-proxy-agent/node_modules/debug/src" attempted to require "supports-color" but could not be resolved, assuming external.
ncc: Module directory "/Users/styfle/Code/zeit/og-image/node_modules/chrome-aws-lambda/source" attempted to require "puppeteer" but could not be resolved, assuming external.
Error: Hash: b222450dedeb92a8e6c4
Version: webpack 5.0.0-alpha.9
Time: 4534ms
Built at: 02/13/2019 9:39:28 AM
                                   Asset       Size  Chunks  Chunk Names
                        Inter-Bold.woff2   93.7 KiB
                     Inter-Regular.woff2   87.1 KiB
             bin/chromium-72.0.3617.0.br   34.2 MiB
                          iltorb/LICENSE   1.13 KiB
       iltorb/build/bindings/iltorb.node    782 KiB
                                index.js   1.12 MiB   {404}  main
          puppeteer-core/CONTRIBUTING.md   8.15 KiB
                  puppeteer-core/LICENSE   11.1 KiB
                puppeteer-core/README.md     18 KiB
             puppeteer-core/package.json   3.28 KiB
                   source/iltorb/LICENSE   1.13 KiB
source/iltorb/build/bindings/iltorb.node    782 KiB
                       source/index.d.ts  399 bytes
Entrypoint main = index.js
 [66] external "fs" 42 bytes {404} [built]
 [77] external "url" 42 bytes {404} [built]
[127] ./src/parser.ts 1.61 KiB {404} [not cacheable] [built]
[298] external "crypto" 42 bytes {404} [built]
[421] external "http" 42 bytes {404} [built]
[426] ./src/file.ts 819 bytes {404} [not cacheable] [built]
[431] external "os" 42 bytes {404} [built]
[444] ./node_modules/puppeteer-core/index.js 1.15 KiB {404} [built]
[589] external "path" 42 bytes {404} [built]
[607] ./src/card.ts 1.3 KiB {404} [not cacheable] [built]
[721] ./src/options.ts 857 bytes {404} [not cacheable] [built]
[898] ./src/chromium.ts 710 bytes {404} [not cacheable] [built]
[940] ./src/template.ts 3.44 KiB {404} [not cacheable] [built]
[963] ./node_modules/marked/lib/marked.js 40.2 KiB {404} [built]
[967] external "util" 42 bytes {404} [built]
    + 145 hidden modules

ERROR in ./node_modules/chrome-aws-lambda/source/index.js 167:6-67
Module not found: Error: Can't resolve '/Users/styfle/Code/zeit/og-image/node_modules/chrome-aws-lambda/source/source/puppeteer/lib' in '/Users/styfle/Code/zeit/og-image/node_modules/chrome-aws-lambda/source'
 @ ./src/options.ts 6:44-72
 @ ./src/chromium.ts 4:18-38
 @ ./src/card.ts 5:19-40

    at compiler.close.n (evalmachine.<anonymous>:1:41090)
    at _promise0.then._result0 (eval at create (evalmachine.<anonymous>:1:813514), <anonymous>:13:1)
    at process._tickCallback (internal/process/next_tick.js:68:7)
</pre>
</details>

## Comments

I'm not sure why this error just started occurring in the `@now/node` builder because `ncc@0.13.0` works and `ncc@0.13.1` works, but the bug occurs in `ncc@0.13.2` and newer.

Here's the build logs for today https://zeit.co/deployments/og-image-1vj25qvla.now.sh/logs

**Comment by guybedford** [comment]
Can you try with the 0.15.0 latest here?

**Comment by styfle** [comment]
The error log I posted is with v0.15.0 (click for full output above)

I just went back to see when the bug was introduced.

**Comment by guybedford** [comment]
Fixed in https://github.com/zeit/webpack-asset-relocator-loader/commit/6122f38cf76d7aafa71d8a221e199ba2881838ee. Will post a loader release shortly with the upgrade.

**Comment by styfle** [comment]
@guybedford Any update on this one?

This is blocking deployments of puppeteer which is something that was being advertised the last 2 weeks.

It's confusing for users 😐 

https://github.com/styfle/og-image/issues/56

**Comment by guybedford** [comment]
@styfle this should be fixed on master.

**Comment by styfle** [comment]
@guybedford Was this fixed in #282?

I didn't realize devDependencies would change the behavior of ncc at runtime.

**Comment by guybedford** [comment]
Yes, we moved the bulk of the analysis logic out into the the separate loader there.

**Comment by styfle** [comment]
Can you publish a new version of ncc with the fix? 😃

**Comment by guybedford** [comment]
I don't have the publish permissions, but I believe you do :)

**Comment by styfle** [comment]
Fixed in ncc@0.15.1
